android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java

```diff
@@ -618,6 +641,41 @@ public void onComplete(@NonNull Task<InstanceIdResult> task) {
                 });
     }
 
+    /*
+     * Unregister your android device with Twilio
+     *
+     */
+
+    @ReactMethod  //
```

Comment:
```suggestion
    @ReactMethod
```

Reply:
Sorry for that) it was a point for me of changes made